package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,1799 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
From a cursory look, it seems fine as just the moved file + PEP8 changes. Unfortunately the relatively unhelpful git diff make this hard to check thoroughly (PEP8 probably could have just been dealt with in a separate PR but too late now).

Is there any particular changes here that you'd like us to look at?

Reply:
As is described in the PR description. I have 

- Fixed the PEP8 issue
- Remove reference to the old HBA in the doc
- Added the code for deprecation related stuff.

None of the code inside the wba is modified.